maxim/filter_objects.py

```
@@ -55,7 +56,8 @@ def parse_incoming_query(incoming_query: str) -> List[RuleType]:
         return []
     operators = ["!=", ">=", "<=", ">", "<", "includes", "does not include", "="]
     result = []
-    for condition in incoming_query.split(","):
+    splitQuery = re.split(r',(?![^\[\]]*\])', incoming_query)
```

Pylint (3.3.7)

[convention] Variable name "splitQuery" doesn't conform to snake_case naming style

(C0103)